src/Symfony/Component/HttpClient/HttpClient.php

```diff
@@ -35,6 +36,8 @@ public static function create(array $defaultOptions = [], int $maxHostConnection
             }
 
             @trigger_error('Configure the "curl.cainfo", "openssl.cafile" or "openssl.capath" php.ini setting to enable the CurlHttpClient', E_USER_WARNING);
+        } elseif (class_exists(ConnectionLimitingPool::class)) {
```

Comment:
should be a `if`, not a `elseif`, so that it is still considered in the case triggering the warning.

Reply:
fixed